cas-server-support-oauth/src/main/java/org/jasig/cas/config/OAuthConfiguration.java

```diff
@@ -51,14 +73,35 @@
     public Config oauthSecConfig() {
         final CasClient oauthCasClient = new CasClient(this.casLoginUrl);
         oauthCasClient.setName(CAS_OAUTH_CLIENT);
+        oauthCasClient.setCallbackUrlResolver((url, context) -> {
```

Comment:
And this is what it looks like with the changes in this PR:

![image](https://cloud.githubusercontent.com/assets/1205228/15054500/a9d835c8-12bc-11e6-97a8-a6315b867f8b.png)